train.py

```diff
@@ -219,7 +315,14 @@ def run_training(args, train_data, val_data):
 
 def main(args):
     tokenizer = AutoTokenizer.from_pretrained(args.model_path, use_auth_token=True)
-    train_dataset, eval_dataset = create_datasets(tokenizer, args)
+    train_dataset, eval_dataset, eval_no_fim = create_datasets(tokenizer, args)
+
+    fim_time = timeit.timeit(lambda: next(iter(eval_dataset)), number=5)
+    non_fim_time = timeit.timeit(lambda: next(iter(eval_no_fim)), number=5)
+
+    print(f"FIM time: {fim_time}")
+    print(f"Non-FIM time: {non_fim_time}")
+
```

Comment:
You didn't observe a slow down in CPU nor GPU, right? If so we can remove this
```suggestion
```

Reply:
Removed